src/data/faq_de.json

```diff
@@ -542,20 +542,107 @@
+                                            { "url" : "#dcc_replacement_issues", "text": "Ich kann ein Zertifikat in der App nicht erneuern. Woran kann das liegen?"},
+                                            { "url" : "#glossary_validity", "text": "Gültigkeit eines EU digitalen COVID-Zertifikats."}
+                                        ]
+                                    }
+                                ]
+                            },
+                            {
+                                "title": "Welche digitalen COVID Zertifikate der EU werden neu ausgestellt?",
+                                "anchor": "dcc_replacement_which",
+                                "active": false,
+                                "textblock": [
+                                    "Folgende Digitale COVID Zertifikateder EU werden neu ausgestellt: <ul> <li>Impfzertifikat mit dem neusten Austellungsdatum</li> <li>Genesenen-Zertifikate</li> </ul>",
+                                    "Folgende Digitale COVID Zertifikate der EU können <strong>nicht</strong> ausgetauscht werden: <ul> <li>Test-Zertifikate</li> <li>Gesperrte Zertifikate</li> </ul>"
+                                ],
+                                "linkList": [
+                                    {
+                                        "title" : "Siehe auch FAQ-Artikel:",
+                                        "links" : [
+                                            { "url" : "#dcc_replacement", "text": "Warum werden Zertifikate neu ausgestellt?"},
+                                            { "url" : "#dcc_replacement_howto", "text": "Wie und wann können Zertifikate in der App neu ausgestellt werden?"},
+                                            { "url" : "#dcc_replacement_vac_status", "text": "Verliere ich meinen Impfstatus wenn eines meiner Zertifikate abläuft?"},
+                                            { "url" : "#dcc_replacement_issues", "text": "Ich kann ein Zertifikat in der App nicht erneuern. Woran kann das liegen?"},
+                                            { "url" : "#glossary_validity", "text": "Gültigkeit eines EU digitalen COVID-Zertifikats."}
+                                        ]
+                                    }
+                                ]
+                            },
+                            {
+                                "title": "Verliere ich meinen Impfstatus, wenn eines meiner Zertifikate abläuft?",
+                                "anchor": "dcc_replacement_vac_status",
+                                "active": false,
+                                "textblock": [
+                                    "Ihr Impfstatus ändert sich durch den Ablauf eines Zertifikats nicht. Sie müssen das entsprechende Zertifikat lediglich in der App neu ausstellen, um Ihren Impfstatus auch in Zukunft über die Corona-Warn-App nachweisen zu können. Folgen Sie hierfür den Anweisungen in der Übersicht Ihrer Zertifikate."

```

Comment:
"in der App neu ausstellen" =>
"in der App neu ausstellen lassen"

Reply:
See English comment.